src/serialport.cpp

```diff
@@ -172,74 +171,73 @@ void EIO_AfterOpen(uv_work_t* req) {
 }
 
 NAN_METHOD(Update) {
-  NanScope();
```

Comment:
found this list of exclusions here

https://github.com/nodejs/nan/blob/4ed41d760313b648b4d212d6ff5374668757be4f/tools/1to2.js#L75


On Tue, Sep 15, 2015 at 2:28 AM, Rod Vagg <notifications@github.com> wrote:

> In src/serialport.cpp
> <https://github.com/voodootikigod/node-serialport/pull/566#discussion_r39463607>
> :
>
> > @@ -172,74 +171,73 @@ void EIO_AfterOpen(uv_work_t* req) {
> >  }
> >
> >  NAN_METHOD(Update) {
> > -  NanScope();
>
> No, anything declared with NAN_METHOD() (or other NAN_*()) doesn't need
> it, that's the point of the text you copy and pasted above.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/voodootikigod/node-serialport/pull/566/files#r39463607>
> .
>